libs/designsystem/src/lib/components/empty-state/empty-state.component.integration.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { MockComponent } from 'ng-mocks';
+
+import { ButtonComponent } from '../button/button.component';
+import { IconComponent } from '../icon/icon.component';
+
+import { EmptyStateComponent } from './empty-state.component';
+
+describe('EmptyStateComponent with slotted buttons', () => {
+  let spectator: SpectatorHost<EmptyStateComponent>;
+  let buttons: ButtonComponent[];
+
+  const createHost = createHostFactory({
+    component: EmptyStateComponent,
+    declarations: [MockComponent(IconComponent), ButtonComponent],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(`
+          <kirby-empty-state
+              iconName="help"
+              title="No items"
+              subtitle="You don't have any items. Call support to add some items to your account."
+          >
+              <button kirby-button attentionLevel='4'>Call support</button>
+              <button kirby-button attentionLevel='3'>Mail support</button>
+              <button kirby-button attentionLevel='2'>Get directions</button>
+              <button kirby-button attentionLevel='1'>Cancel</button>
```

Comment:
We generally use `"` for property assignment:
```suggestion
              <button kirby-button attentionLevel="4">Call support</button>
              <button kirby-button attentionLevel="3">Mail support</button>
              <button kirby-button attentionLevel="2">Get directions</button>
              <button kirby-button attentionLevel="1">Cancel</button>
```